tests/integration/targets/test_azure_virtual_machine_with_public_ip/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+azure_virtual_machine_with_public_ip_vm_name: "{{ resource_prefix }}-vm"
+azure_virtual_machine_with_public_ip_vm_image:
+  offer: RHEL
+  publisher: RedHat
+  sku: 8-LVM
+  version: latest
+azure_virtual_machine_with_public_ip_vm_username: 'azureuser'
+azure_virtual_machine_with_public_ip_vm_admin_password: 'Password123!'
```

Comment:
I don't think we should provide a default password.  We should force the user to specify one.

Reply:
fixed